### PR TITLE
scope launch script improvements

### LINKED ISSUE #1556: Scope command line provides inconsistent documentation re launch arguments
I'm trying to launch scope, pointing it at a weave net instance.

If I run `scope help`, I get a swathe of documentation about options, but nothing about arguments. However, if I type something invalid (e.g. `scope mcduck`), I get actual usage information:

```
Usage:
scope launch [<peer> ...]
scope stop
scope command

scope <peer>    is of the form <ip_address_or_fqdn>[:<port>]
```

This tells me what arguments `launch` takes.

I would expect to see these arguments also documented in `scope help`

### LINKED ISSUE #2073: 'scope help' does not appear in usage
```
$ scope boo
Unknown scope command 'boo'
Usage:
scope launch [<peer> ...]
scope stop
scope command

scope <peer>    is of the form <ip_address_or_fqdn>[:<port>]
$ scope help
Usage:

scope help     - Print this

scope launch   - Launch Scope
Usage of /home/weave/scope:
  -alsologtostderr
    	log to standard error as well as files
  -app-only
    	Only run the app
...
```

Related:
- `--help` should ideally work too
- we have *two*, inconsistent, pieces of usage info in the script - one in `usage()` and one in the `help` handler
- the help handler is broken - it's actually supposed to output all script commands (like `usage()`) but with the launch section expanded to show all options, but because of invalid nested usage of 'here' doesn't display all commands.

### PR DESCRIPTION
**Pull request by ekimekim**
* fix linter issues
* make some attempt at correct quoting in `scope command` when available
* fix and rewrite usage/help text for clarity and extra info
* don't print container id as it's confusing when presented out of context, and rarely useful

See individual commits for discussion on each point.

Fixes #2073. Fixes #1556. Fixes https://github.com/weaveworks/scope/issues/1557. Fixes https://github.com/weaveworks/scope/issues/1560.

### CODE REVIEW

**Comment by rade** [comment]
This fixes #2073, right? And perhaps some others?

**Comment by ekimekim** [comment]
yes, sorry. I edited to mention that between you loading and you commenting.
I reviewed other open issues, I'd say this also fixes https://github.com/weaveworks/scope/issues/1556 which was practically a duplicate, but nothing else.

**Comment by ekimekim** [comment]
Slightly changed the wording to properly fix #1556

**Comment by ekimekim** [comment]
Also now fixes https://github.com/weaveworks/scope/issues/1557 and https://github.com/weaveworks/scope/issues/1560. https://github.com/weaveworks/scope/issues/1560 partially addresses https://github.com/weaveworks/scope/issues/1562

File: scope [hunk 1]
```diff
@@ -172,40 +186,32 @@ check_docker_version
 
 case "$COMMAND" in
     command)
-	# TODO: properly escape/quote the output of "$@"
-        echo $(launch_command) "$@"
+        # Most systems should have printf, but the %q specifier isn't mandated by posix
+        # and can't be guarenteed. Since this is mainly a cosmetic output and the alternative
```
Comment (jml) [inline scope]: Nit: guaranteed

Comment (rade) [inline scope]: @jml the issues are listed in the PR summary.

Comment (jml) [inline scope]: Ah yes. Thanks.

**Pull request merged**